roles/smoker/tasks/main.yml

```
@@ -4,6 +4,23 @@
   package:
     name: '{{ smoker_browser_packages }}'
 
+- name: 'Install podman'
+  become: true
+  package:
+    name: 'podman'
+
+- name: Set max user namespaces
+  become: true
+  shell: echo 100000 > /proc/sys/user/max_user_namespaces
```

Isn't this just `user.max_user_namespaces` that can be set via sysctl?
```suggestion
  sysctl:
    name: user.max_user_namespaces
    value: 100000
```